### DISCUSSION TITLE
QDate with Locale Property

### DISCUSSION THREAD

**Discussion by fwiwDev** [quasarframework/quasar#9099] [General - Components / Directives / etc] [answered]
Hi,

I tried to use QDate with locale but can't seem to figure it out. Has anyone had success with this?

I used the locale formatting option for QDate here: (https://quasar.dev/vue-components/date#qdate-api)

Please see my CodePen here: (https://codepen.io/quasy/pen/KKaLQBB?editors=101)

Thanks for taking a look.

-Q

**Comment by ontwikkelfabriek** [comment, marked as answer]
That's because **if** you use a custom `locale` you will have to define all properties like this:

https://codepen.io/ontwikkelfabriek/pen/abpgOMe

**Reply by fwiwDev** [reply]
Thanks so much!
